src/view.js

```diff
@@ -15,6 +15,24 @@ class View {
         wrapperSimulator.classList.add('wrapper__simulator');
         wrapper.append(wrapperSimulator);
 
+        const simInputAge = this.createInput({placeholder: 'Min age', id: 'minAge', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputAge);
+
+        const simInputHealth = this.createInput({placeholder: 'Min Health', id: 'minHealth', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputHealth);
+
+        const simInputOffense = this.createInput({placeholder: 'Max Offense', id: 'maxOffense', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputOffense);
+
+        const simInputMoneyMin = this.createInput({placeholder: 'Min Money', id: 'minMoney', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputMoneyMin);
+
+        const simInputMoneyMax = this.createInput({placeholder: 'Max Money', id: 'maxMoney', classes: ['simulator__input']});
+        wrapperSimulator.append(simInputMoneyMax);
+
+        const simButton = this.buttonCreate('simulate', 'Simulate');
+        wrapperSimulator.append(simButton);
```

Comment:
why so much append to WrapperSimulator? please use one append and one const simInputs = this.createInput({placeholder: [1,2,3,4,...] , id: [1,2,3,4...], class: [1,2,3,4...], require: [true, false...]};